patches/minecraft/net/minecraft/client/renderer/texture/Stitcher.java.patch

```diff
@@ -22,16 +22,15 @@
              throw new StitcherException(stitcher$holder.field_229213_a_, list.stream().map((p_229212_0_) -> {
                 return p_229212_0_.field_229213_a_;
              }).collect(ImmutableList.toImmutableList()));
-@@ -103,7 +111,7 @@
-          boolean flag4 = flag2 && j != l;
+@@ -104,6 +112,7 @@
           boolean flag;
           if (flag3 ^ flag4) {
--            flag = flag3;
+             flag = flag3;
```

Comment:
Why?

Reply:
It looks like an unnecessary deletion to me. The assignment just gets overwritten on the next line.